### ISSUE TITLE
2.9.4 Patch Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3197]
- #3189
- #3193
- #3194
- #3187
- #3195

## Pick list
```
git cherry-pick 7514c9f1a354ec19a0a1ac9895ae4244f6b4dc18
git cherry-pick ac6a3763f256203b57804dcd15faa7863f43fbf4
git cherry-pick 3ddf38e3a239ae8656416c21d336c436e4ab66bd
git cherry-pick 9bc53a3d1ddef57c3558de0548bc5de443e84972
git cherry-pick 9885c9a42dabe52fef6240448fd6b9152d8315f3
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.9.3..HEAD --pretty='format:- %s'`
```
_Note: This release contains an update for the Antibot module that addresses a critical issue that was preventing the module from blocking spam submissions._

- Closes #3190: Add core patch for BigPipe Javascript error(s) on node add/edit pages. (#3195)
- Closes #3185 Workbench Access menu schemes may cause menu items to change when pages are saved (#3187)
- Close #3192 Update patch for drupal.org/i/2552541 (#3194)
- Closes #3191 Remove patch for drupal.org/i/3163925 (#3193)
- Update drupal/antibot requirement from 2.0.2 to 2.0.3 (#3189)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.9.4
